LDM-753.tex

```diff
@@ -1,60 +1,105 @@
+\usepackage{enumitem}
+\usepackage{booktabs}
+\usepackage{arydshln}
+\usepackage{afterpage}
+\usepackage{pdflscape}
+\usepackage{graphicx}
+\usepackage{lipsum}
 
-% Generated by Makefile
-\input{meta}
 
-% Package imports go here.
+\input{meta.tex}
 
-% Local commands go here.
+%\input{aglossary.tex}
+%\makeglossaries
 
-% If you want glossaries, uncomment:
-% \input{aglossary.tex}
-% \makeglossaries
+\DeclareRobustCommand{\cmdkey}{\raisebox{-.035em}{\includegraphics[height=.75em]{figures/cmdkey}}}
 
-\title{Vera C. Rubin Observatory DM Services Verification Document}
-% \setDocSubtitle{Optional subtitle}
+\setcounter{tocdepth}{2}
 
-\author{%
-Gabriele Comoretto
-}
+\begin{document}
+
+\providecommand{\tightlist}{%
+  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
+
+\def\product{LSST Data Management}
+
+\setDocCompact{true}
+
+\title[DM Infrastructure Verification Document]{Vera C. Rubin Observatory DM Science Verification Document}
 
-\setDocRef{LDM-753}
-\setDocUpstreamLocation{\url{https://github.com/lsst/LDM-753}}
-\date{\vcsDate}
-% \setDocCurator{The Curator of this Document}
+\author{Jeff Carlin}
+\setDocRef{\lsstDocType-\lsstDocNum}
+\setDocDate{\vcsDate}
 
-\setDocAbstract{%
-Add abstract text.
+\setDocAbstract {
+Data Management infrastructure Verification Elements Baseline.
 }
 
-% Revision history.
-% Order: oldest first.
-% Fields: VERSION, DATE, DESCRIPTION, OWNER NAME.
-% See LPM-51 for version number policy.
+% Most recent last
 \setDocChangeRecord{%
-  \addtohist{1}{2020-08-11}{Unreleased.}{Gabriele Comoretto}
+	\addtohist{}{2020-08-11}{First draft}{}
 }
 
-\begin{document}
+\setDocUpstreamLocation{\url{https://github.com/lsst/ldm-753}}
+\setDocUpstreamVersion{\vcsRevision}
 
 \maketitle
 
-% ADD CONTENT HERE
-% You can also use the \input command to include several content files.
 
-\appendix
+\section{Introduction}\label{sec:intro}
+
+
+\subsection{Scope}\label{sec:scope}
+
+The scope of this document is to capture the content and details of all DM Verification Elements
+categorized in the \textbf{Service} sub-component. This will make possibile:
+
+\begin{itemize}
+\item provide to users and stakeholders the verification elements details, without the need to access Jira
+\item approve changes to the verification elements
+\end{itemize}
+
+\subsection{Specification Flow-down}\label{sec:sepcflowd}
+
+
+\subsection{LSST Verification and Validation JIRA Project (LVV)}\label{sec:lvv}
+
+The LSST Verification and Validation JIRA Project lists the specifications within or derived from, 
+and traceable to, the DMSR specifications, in Verification Elements that also specify the methods 
+to be used to verify, the responsible parties, and additional notes regarding verification, per the 
+\citeds{LSE-160} LSST Verification and Validation Process. The Verification Elements then have one or more 
+Test Cases associated with them that describe the implementation of the verification activities in terms 
```

Comment:
The LSST Verification and Validation JIRA Project contains the detailed specifications within or derived from,
and traceable to, the DMSR specifications, in Verification Elements.
Verification Elements also specify the verification methods, the responsible parties, and additional notes regarding verification, as per the \citeds{LSE-160} LSST Verification and Validation Process.
\citeds{LSE-160} LSST Verification and Validation Process.